### DISCUSSION TITLE
Deployment Queued in 1 day

### DISCUSSION THREAD

**Discussion by antechinsight** [vercel/vercel#5080] [Help] [answered]
Hello, 

I have deploy on Vercel via Github App.
But deploy with status `Queued` in 1 day.

Please help. Thanks.

**Comment by styfle** [comment, marked as answer]
Hi @antechinsight 

If you exceed your concurrency limit, then subsequent deployments get queued. You can cancel an ongoing deployment by visiting the [Dashboard](https://vercel.com/dashboard) and selecting your project, then your deployment, then clicking the 3 dots next to the Visit button, then Cancel. Read more: [Why are my Vercel builds queued?](https://vercel.com/knowledge/why-are-my-vercel-builds-queued)

If you are still seeing Queued deployments, please [Submit a Support Request](https://vercel.com/support/request) and we'll take a look.
